src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final int MAX_VALUE = Color.values().length;
+    private final int indexOfColor = new Random().nextInt(MAX_VALUE);
```

Comment:
this should be calculated every time method `getRandomColor` is being called, so move it to the method, also rename variable just to `index`
```suggestion
```